pastis/util.py

```diff
@@ -676,3 +676,10 @@ def find_repo_location(package='pastis'):
     :param package: string, name of package within the repository whose path we are looking for
     """
     return os.path.abspath(os.path.join(find_package_location(package), os.pardir))
+
+
+def continous_dm_coo(nbactuator, segment):
```

Comment:
Let's rename this function to something more obvious (what does "coo" mean?) and please add a docstring.